validator/bin/reflector

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+ROOT=$(realpath $(dirname $0)/../..)
+
+if [[ $# < 4 ]]; then
+    echo Usage: $0 site_dir project_id device_id directive [directives...]
+    echo "  Directive is something like update/config:sites/udmi_site_model/devices/AHU-1/out/generated_config.json"
+    false
```

Comment:
exit 1?

Reply:
same as other comment... not sure if there's a clear reason for one over the other because they're just "different" in how they handle different edge use cases...